applications/HDF5Application/tests/test_hdf5_output_process.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.HDF5Application as KratosHDF5
+
+import hdf5_output_process
+
+class TestHDF5OutputProcess(KratosUnittest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_serial_hdf5_output(self):
```

Comment:
For consistency
```suggestion
     def test_HDF5SerialOutput(self):
```
probably we will need to come up with a less general name for the test.